README.md

```diff
@@ -1,63 +1,80 @@
-## Webpack Virtual Modules
+# Webpack Virtual Modules
 
 [![Build Status](https://travis-ci.org/sysgears/webpack-virtual-modules.svg?branch=master)](https://travis-ci.org/sysgears/webpack-virtual-modules)
 [![Twitter Follow](https://img.shields.io/twitter/follow/sysgears.svg?style=social)](https://twitter.com/sysgears)
 
-Webpack Plugin that allows dynamical generation of in-memory virtual modules. Watch mode is supported. Any write to virtual module will be seen by Webpack as if the corresponding file was changed.
+**Webpack Virtual Modules** is a plugin that allows for dynamical generation of in-memory virtual modules for JavaScript 
```

Comment:
Why `allows for` ?

Reply:
Because we don't use an object after "allow".
You can write "allow + object + to do", "allow + -ing" (which is discouraged), or "allows for + noun".

PS: BTW, it should be "dynamic" not "dynamical".